bower.json

```diff
@@ -1,30 +1,53 @@
 {
   "name": "lazy-imports",
+  "version": "2.0.0",
```

Comment:
In the docs atm I believe it specifies using `^2.0.0` when installing polymer 2 compatible elements from PolymerElements.